.github/workflows/run-code-checks.yaml

```diff
@@ -15,24 +15,20 @@ jobs:
       - name: Checkout 🛎️
         uses: actions/checkout@v4
 
-      - name: Set up PDM 🗻
-        # using the PDM action because actions/setup-python doesn't support
-        # caching for PDM out of the box
-        # https://github.com/pdm-project/setup-pdm?tab=readme-ov-file#why-do-i-need-this-action
-        uses: pdm-project/setup-pdm@v4
+      - name: Set up Python 🐍
+        uses: actions/setup-python@v5
```

Comment:
When there's no Python version explicitly supplied, the `setup-python` action uses the version specified in `.python-version`